snetd/cmd/memory_storage_util.go

```diff
@@ -0,0 +1,109 @@
+package cmd
```

Comment:
This file looks like absolute copy of `escrow/memory_storage.go`, you can remove it and use `escrow/memory_storage.go` instead. If the problem is how to use private `memoryStorage` struct then answer is: you can use `AtomicStorage` interface instead. `memoryStorage` implements this interface.